### DISCUSSION TITLE
Have you seen this document for the Trifo M6

### DISCUSSION THREAD

**Discussion by 001Tycoon** [VictorDrijkoningen/trifo-robotics-rev-eng#3] [General]
[Ethical Hacking of a Robot Vacuum Cleaner](https://kth.diva-portal.org/smash/get/diva2:1450590/FULLTEXT01.pdf)

**Comment by VasanK16** [comment]
I have been using Trifo Ironpie m6 from 2020 June and it works great but suddenly from Jan 2025 I lost the access to the app
